.pre-commit-config.yaml

```diff
@@ -10,6 +10,7 @@ repos:
       - id: check-ast
       - id: trailing-whitespace
       - id: check-merge-conflict
+        exclude: AUTHORS
```

Comment:
I don't think this is correct. If you are rebasing and there's a merge conflict, fix the conflict, don't exclude the test. Please revert this change.

Reply:
No, because there is no merge conflict here. Check for the merge conflicts detects "=======" string from AUTHORS file as a leftover from an unresolved conflict, but it is a regular AUTHORS file content, see AUTHORS on current master: https://github.com/jazzband/django-oauth-toolkit/blob/6017f079abc0da9f66fb0082539a88ce56b03a42/AUTHORS#L2 Without changing checks I can't commit with changes in AUTHORS file.

Reply:
Oh yeah, I see that now. Wow.